sdk/identity/identity/src/credentials/authFileCredential.browser.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { AccessToken, TokenCredential, GetTokenOptions } from "@azure/core-http";
+import { TokenCredentialOptions } from "../client/identityClient";
+
+const BrowserNotSupportedError = new Error("AuthFileCredential is not supported in the browser.");
+
+export class AuthFileCliCredential implements TokenCredential {
```

Comment:
Shouldn't the class name be `AuthFileCredential` as it is in the non-browser implementation?